src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -869,6 +869,8 @@ public void SetInputRegion(Rectangle inputRegion)
         /// </summary>
         /// <param name="level">The notification window level.</param>
         /// <returns>True if no error occurred, false otherwise.</returns>
+        /// <privilege>http://tizen.org/privilege/window.priority.set</privilege>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
```

Comment:
"can be due to..." -> "can be thrown due to..." will sound better, if you decide to change it then change all occurrences

Reply:
@wiertel 
Hello, thank you for the review.
I have changed by following your comment.
please review again. thank you.